packages/react-components/src/components/Heading.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import cx from 'classnames';
+
+enum sizeToElement {
+  xl = 'h1',
+  lg = 'h2',
+  md = 'h3',
+  sm = 'h4',
+  xs = 'h5',
+}
+
+type TSize = 'xl' | 'lg' | 'md' | 'sm' | 'xs';
```

Comment:
Also: how about using `enums` instead of `strings`? Less chance for a typo
```
type HeadingSize = SizeMark.XL | SizeMark.LG | SizeMark.MD | ..... ;
```
Although I would like the `size` property to remain string, e.g. `<Heading size="xs" />` not `<Heading size={SizeMark.XS} />`

Reply:
> The convention to use `T` prefix for types feels 1990s to me although I know some people like it

Not sure what better here, but seems like a good point.